apps/dashboard/src/components/workflow-editor/steps/email/views/maily-variables-list-view.tsx

```diff
@@ -71,6 +74,10 @@ export const MailyVariablesListView = React.forwardRef(
           digestStepName: digestStepBeforeCurrent?.stepId,
         });
         selectedItem = { ...selectedItem, name: value };
+
+        track(TelemetryEvent.DIGEST_VARIABLE_SELECTED, {
+          type: item.name,
+        });
```

Comment:
Is it possible to have a generic "Variable Selected" event and pass the name as a pram?

Reply:
Yes, do we want to track all the variables including `payload`, `subscriber` etc
or just the digest ones for now?